initial_setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+URL="http://0.0.0.0:8000/"
+status_code=000
+ENV_EXAMPLE=".env.example"
+ENV_ORIG=".env"
+#TIMEOUT_TIME=60
+
+exit_on_error () {
+    echo "FAILED AT: $@"
+    exit
+}
```

Comment:
This might be simpler:

```
function wrap {
  # Echo what the command is before running it
  echo "$@"
  # Run the passed command
  eval "$@"
  local eval_status=$?
  # Enable panicing this function's parent in case of non-zero exit
  set -e
  # Do non-zero exit, if eval's return was non-zero
  [ $eval_status -eq 0 ] || (echo "failed" && exit 1)
  # Disable panicing this function's parent in case of non-zero exit
  set +e
}

```

Usage: 

```
wrap docker exec mathesar_service sh -c "python manage.py migrate && python install.py"
```

`wrap` might not be the best name, feel free to change it.